packages/diagrams/lib/src/custom_list_item.dart

```diff
@@ -0,0 +1,137 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'diagram_step.dart';
+
+class _VideoDescription extends StatelessWidget {
+  const _VideoDescription({
+    Key key,
+    this.title,
+    this.user,
+    this.viewCount,
+  }) : super(key: key);
+
+  final String title;
+  final String user;
+  final int viewCount;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(5.0, 0.0, 0.0, 0.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Text(
+            title,
+            style: TextStyle(
+              fontWeight: FontWeight.w500,
+              fontSize: 14.0,
+            ),
+          ),
+          const Padding(padding: EdgeInsets.symmetric(vertical: 2.0)),
+          Text(user, style: TextStyle(fontSize: 10.0)),
+          const Padding(padding: EdgeInsets.symmetric(vertical: 1.0)),
+          Text(
+            '$viewCount views',
+            style: TextStyle(fontSize: 10.0),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+/// A sample list item that looks similar to a YouTube related video item.
+///
+/// Since users are misusing ListTile material widget, this example was created
+/// to help provide inspiration for alternative ways to create list items.
+class CustomListItem extends StatelessWidget {
+  const CustomListItem({
+    this.user,
+    this.viewCount,
+    this.title,
+  });
+
+  final String user;
+  final int viewCount;
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 5.0),
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          const Expanded(child: FlutterLogo(size: 96), flex: 2),
```

Comment:
Nit: Formatting per style guide across this PR.
See: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting
In particular, multi-line arguments and trailing commas.